pkg/signup/signup.go

```diff
@@ -7,6 +7,8 @@ type Signup struct {
 	ConsoleURL string `json:"consoleURL,omitempty"`
 	// The Che Dashboard URL of the cluster which the user was provisioned to
 	CheDashboardURL string `json:"cheDashboardURL,omitempty"`
+	// The server api URL of the cluster which the user was provisioned to
+	ApiEndpoint string `json:"apiEdnpoint,omitempty"`
```

Comment:
typo here !!!!

Reply:
Good catch @jeffmaury!!!
```suggestion
	ApiEndpoint string `json:"apiEndpoint,omitempty"`
```

Reply:
done